Yoast/wordpress-seo
# Adds the breadcrumb schema to the homepage

## Context


* All details in here: https://github.com/Yoast/wordpress-seo/issues/13325

## Summary


This PR can be summarized in the following changelog entry:

* Adds the breadcrumb Schema to the homepage.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps: (edited by @johannadevos)

* Before you start testing, make sure that breadcrumbs are enabled and visible.

**To reproduce the problem**
* Without this PR, when you inspect the page source for the homepage, there is no `BreadcrumbList` in the Schema output.

**To test the fix**
* Inspect the page source for the homepage again.
* Schema should show one breadcrumb:

```json
	        {
	            "@type": "BreadcrumbList",
	            "@id": "http://basic.wordpress.test/#breadcrumb",
	            "itemListElement": [
	                {
	                    "@type": "ListItem",
	                    "position": 1,
	                    "item": {
	                        "@type": "WebPage",
	                        "@id": "http://basic.wordpress.test/",
	                        "url": "http://basic.wordpress.test/",
	                        "name": "Home"
	                    }
	                }
	            ]
	        }
```

* You should see this output in all of the following scenarios:
    * A non-static homepage ("Your latest posts" in the Reading settings).
    * A static home page, set to a non-nested page.
    * A static home page, set to a nested page.

## UI changes

## Documentation

## Quality assurance

* [x] I have added unittests to verify the code works as intended (by @johannadevos).

Fixes https://github.com/Yoast/wordpress-seo/issues/13325
